spec/factories/organizations.rb

```diff
@@ -56,6 +56,20 @@
       deadline_day { nil }
     end
 
+    trait :with_items do
+      after(:create) do |instance, evaluator|
+        BaseItem.seed_items if BaseItem.count.zero?
+        Organization.seed_items(instance)
+      end
+    end
+
+    trait :with_items do
+      after(:create) do |instance, evaluator|
+        BaseItem.seed_items if BaseItem.count.zero?
+        Organization.seed_items(instance)
+      end
+    end
```

Comment:
Isn't this the same thing twice?

Reply:
It isn't `Organization.seed_items` (specifically the class method) seeds `BaseItem.all` to either the specified org or to all orgs. Ask me how I know :upside_down_face:.